projects/ccf-eui/src/styles.scss

```diff
@@ -11,6 +11,7 @@
 @use 'themes/gtex_dark';
 
 @import 'swiper/swiper.min.css';
+@import "~@angular/material/prebuilt-themes/indigo-pink.css";
```

Comment:
This doesn't seem quite right... Why has this been added?

Reply:
The toggle button styles remain consistent with this import. If we remove this import the borders and certain styles are not visible for non-selected buttons.

Reply:
Then you have a larger problem that you don't understand.